packaging_ignore.yml

```diff
@@ -9,6 +9,8 @@ base:
   - "gpg: WARNING: options in `/root/.gnupg/gpg.conf' are not yet active during this run"
   - ".* warning: overriding recipe for target 'check'"
   - "WARNING generated by debuild:"
+  - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
+  - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
```

Comment:
I'm not sure what those are about, but this is fine since we ignore them for citus packages as well.